diff --git a/dist/node/coverage-reporter.js b/dist/node/coverage-reporter.js
index b3fe10454fefd2206ff3fda99c62cf4417359697..5e998a1e65e8bbc2bd37342d7817da424e195757 100644
--- a/dist/node/coverage-reporter.js
+++ b/dist/node/coverage-reporter.js
@@ -1351,5 +1351,6 @@ var import_istanbul_lib_report = __toESM(require_istanbul_lib_report(), 1), Stor
   }
 };
 export {
+  StorybookCoverageReporter as default,
   StorybookCoverageReporter as "module.exports"
 };
