diff --git a/dist/client.js b/dist/client.js
index 327d6f77f26996468f56e799b6c0f35a0b39d643..789f5fcec49ced1ff9bd95b2f007251a5956e838 100644
--- a/dist/client.js
+++ b/dist/client.js
@@ -7,6 +7,7 @@ exports.getMDXComponent = getMDXComponent;
 exports.getMDXExport = getMDXExport;
 var React = _interopRequireWildcard(require("react"));
 var _jsx_runtime = _interopRequireWildcard(require("react/jsx-runtime"));
+var _jsx_dev_runtime = _interopRequireWildcard(require("react/jsx-dev-runtime"));
 var ReactDOM = _interopRequireWildcard(require("react-dom"));
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
@@ -37,7 +38,7 @@ function getMDXExport(code, globals) {
   const scope = {
     React,
     ReactDOM,
-    _jsx_runtime,
+    _jsx_runtime: process.env.NODE_ENV === 'production' ? _jsx_runtime : _jsx_dev_runtime,
     ...globals
   };
   // eslint-disable-next-line
diff --git a/dist/index.js b/dist/index.js
index b4e645e66705edac1a03c04513a3c32c8eeebcfc..9813d75dc0bbb6ac444a458f04e562ce8b212d86 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -13,6 +13,7 @@ var esbuild = _interopRequireWildcard(require("esbuild"));
 var _nodeResolve = require("@esbuild-plugins/node-resolve");
 var _esbuildPluginGlobalExternals = require("@fal-works/esbuild-plugin-global-externals");
 var _uuid = require("uuid");
+const { type } = require("os");
 var _dirnameMessedUp = _interopRequireDefault(require("./dirname-messed-up.cjs"));
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
@@ -196,6 +197,14 @@ async function bundleMDX({
       'react/jsx-runtime': {
         varName: '_jsx_runtime',
         type: 'cjs'
+      },
+      'react/jsx-dev-runtime': {
+        varName: '_jsx_runtime',
+        type: 'cjs'
+      },
+      'react/compiler-runtime': {
+        varName: '_react_compiler_runtime',
+        type: 'cjs'
       }
     }),
     // eslint-disable-next-line new-cap
